src/socket-gas-helpers/estimateGasLimit.ts

```diff
@@ -0,0 +1,36 @@
+import { BigNumber, providers, utils } from "ethers";
+import PlugABI from "@socket.tech/dl-core/artifacts/abi/IPlug.json";
+
+import { ChainDetails, Inputs, getPayload } from "./utils";
+import { getArbitrumGasLimitEstimate } from "./arb-estimate";
+import { getOpAndEthGasLimitEstimate } from "./op-n-eth-estimate";
+
+export const estimateGasLimit = async (
+  chainDetails: ChainDetails,
+  inputs: Inputs,
+  withoutHook?: boolean
+): Promise<BigNumber> => {
+  const srcChainSlug = chainDetails.srcChainSlug;
+
+  const provider = new providers.StaticJsonRpcProvider(chainDetails.dstRPC);
+  const payload = await getPayload(inputs, provider, withoutHook);
```

Comment:
we dont need to handle withoutHook case, aevo and lyra dont need this sdk and version is being deprecated